.github/workflows/pull-request-management.yml

```diff
@@ -90,6 +91,14 @@ jobs:
             pyavd:
               - 'python_avd/*'
               - 'python_avd/**/*'
+            anta_runner:
+              - 'ansible_collections/arista/avd/roles/anta_runner/*'
```

Comment:
I feel like whenever we change something in eos_designs we should re run this to make sure how it impacts the catalog

Reply:
ok I should read the full file.